addons/account/models/account_tax.py

```diff
@@ -105,6 +105,7 @@ def _default_tax_group(self):
     sequence = fields.Integer(required=True, default=1,
         help="The sequence field is used to define order in which the tax lines are applied.")
     amount = fields.Float(required=True, digits=(16, 4), default=0.0)
+    real_amount = fields.Float(string='Real amount to apply', compute='_compute_real_amount', store=True)
```

Comment:
This field is useless....
- Used only for POS but can be replaced by a single multiplication
- Part of no business flow
- No review asked to the accounting team
- No migration script for that
Come on guys...

Reply:
It was here as a fix in a function since 2019 https://github.com/odoo/odoo/pull/36140, because you changed the taxes and broke the POS tax computation with your repartition lines. It is used in front and was set as a field to get it easily when we need to compute tax.

Just missing the migration script, my bad, but all other reasons are wrong, and it does not impact the accounting stuff so I don't really see why we would have need your opinion.

I doubt that all your migration scripts are there and perfect too.

Reply:
> Part of no business flow

Just FYI, it is sent to and used in the PoS UI during the loading thanks to `load_pos_data` and `_pos_ui_models_to_load`. Hence why you couldn't find any occurrences of  `_get_pos_ui_account_tax`

Reply:
@smetl BTW we should probably use it on the sale orders and the invoices to avoid having those kind of inconsistencies when the repartition lines are not fullfulling the amount.
 
 
![image](https://user-images.githubusercontent.com/20767708/198614249-7ca26c95-8feb-414e-8b1a-2a9ad397a81b.png)

Reply:
@pimodoo The repartition lines are well managed python-side. This is absolutely not related to the unconsistency with sale orders.
This field is useless because you can perform yourself the computation of your `real_amount` (that doesn't mean anything btw...) in `_get_pos_ui_account_tax`.
And sorry but you **need** to ask a review for that. We are challenged for any single field we add. This is not up to you to decide that yourself.

Reply:
The real amount, is the effective percentage that you have to apply on your base price to know your total price.